### PR TITLE
Update test posts

### PR DESCRIPTION
**Pull request by SukkaW**
- Add a markdown style test: The test post covers nearly all the usages of markdown.
- Add a code highlight test: The test posts includes 16 code samples from `highlight.js` demo.
- Update elements test post: Add `<kbd>` html tag.

### CODE REVIEW

File: source/_posts/markdown.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+
+## Links
+
+```markdown
+[I'm an inline-style link](https://www.google.com)
+
+[I'm an inline-style link with title](https://www.google.com "Google's Homepage")
+
+[I'm a reference-style link][Arbitrary case-insensitive reference text]
```
Comment (curbengh) [inline source/_posts/markdown.md:157]: need to be rendered, not even github render this.

File: source/_posts/markdown.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+[I'm a relative reference to a repository file](../blob/master/LICENSE)
+
+[You can use numbers for reference-style link definitions][1]
```
Comment (curbengh) [inline source/_posts/markdown.md:161]: this one too.

File: source/_posts/markdown.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+Some text to show that the reference links can follow later.
+
+
+## Images
+
+```markdown
+hover to see the title text:
+
+Inline-style:
+![alt text](https://i1.wp.com/theme-suka.github.io/demo/img/suka-favicon.png "Logo Title Text 1")
```
Comment (curbengh) [inline source/_posts/markdown.md:189]: something more neutral? e.g. https://hexo.io/icon/favicon-196x196.png

**Review by NoahDragon — APPROVED** [review]
👍

**Review by curbengh — COMMENTED** [review]
some lines are not rendered.

**Pull request merged**